src/python/DQM/GUI.py

```diff
@@ -1181,6 +1181,14 @@ def sessionSetJsonmode(self, session, *args, **kwargs):
     self.gui._saveSession(session)
     return self._state(session)
 
+  def sessionSetJSrootmode(self, session, *args, **kwargs):
+    jsrootmode = kwargs.get('mode', None)
+    if isinstance(jsrootmode, str) or jsrootmode in ("yes", "no"):
```

Comment:
You're right it's unnecessary. Removed it from both  sessionSetJSrootmode and sessionSetJsonmode